imap_processing/swapi/swapi_utils.py

```diff
@@ -8,11 +9,13 @@
 
 
 class SWAPIAPID(IntEnum):
-    """Create ENUM for apid.
+    """
+    Create ENUM for apid.
 
-    Parameters
+    Attributes
     ----------
-    IntEnum : IntEnum.
+    IntEnum : IntEnum
+        No idea todo Check.
```

Comment:
This isn't an Attribute, it's a [parent class](https://www.w3schools.com/python/python_inheritance.asp) so I think you can remove it..? If I remember, Tenzin added this because the docs were failing. So, try removing it, and if the docs start failing, then you can add it in but say "This is a parent class, not an attribute" basically.

Incidentally, an IntEnum is a way to create an Enum in Python, which is basically like a constant set of key-value pairs. If you have a known pair, for example that SWP_HK APID is always 1184, you can make an enum and then use SWP_HK_APID in your code (very clear) instead of 1184 (very unclear).